cmd/continuity/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/continuity/cmd/continuity
 
-go 1.17
+go 1.19
```

Comment:
?

Reply:
It uses a replace rule for the main module (`github.com/containerd/continuity`), and the main module requires go 1.19 as minimum. https://github.com/containerd/continuity/blob/898d7ede043bb5be87b5a44d1e1b1e4289a7e125/go.mod#L1-L3